agent/agent_test.go

```diff
@@ -2483,7 +2483,7 @@ func (s *AgentSuite) TestUpsertCheckRun() {
 	s.tc.taskConfig.Task.Requester = evergreen.GithubPRRequester
 
 	s.tc.taskConfig.Expansions.Put("checkRun_key", "checkRun_value")
-	checkRunOutput, err := buildCheckRun(s.ctx, s.tc)
```

Comment:
I was wondering if there was a reason we had it because buildCheckRun doesn't seem to be using ctx. I'll put it back in just in case we want to pass down context for tracing though